### ISSUE TITLE
Accessibility text missing from feed link

### ISSUE THREAD

**Issue by ShaneCurcuru** [benbalter/retlab#9]
Running the http://wave.webaim.org/ tool on my site using an unmodified retlab theme gives an accessibility error for the lack of any visible text or alternate description for the feed.xml link.
